jina/parsers/flow.py

```diff
@@ -52,6 +52,12 @@ def mixin_k8s_parser(parser):
         if _SHOW_ALL_ARGS
         else argparse.SUPPRESS,
     )
+    gp.add_argument(
+        '--k8s-startup-time',
+        type=int,
+        default=60,
+        help='Time to wait until the kubernetes flow started. In the future, we will solve this using probes.',
```

Comment:
Do not add this, this will be soon solved

Reply:
I have to add this otherwise the code will not work.
Let's have the independent feature development for probes and port-forward.
This attribute can be removed when introducing probes.

Reply:
No, this will wait until the other, they are part of the same, do not want to add an unnecessary attribute in the wrong direction. The other PR will be ready soon